source/designers-guide/snippets/index.md

```diff
@@ -59,6 +59,18 @@ While snippets are mostly used to translate plain text, they are flexible enough
 {s name="frontend/checkout/cart/separate_dispatch"}<strong>bold example text</strong>{/s}
 ```
 
+### Using dynamic snippets
+
+You can create snippets, that use dynamic snippet names. This can be of use, when you only have a value/name stored in a variable, but want a translation of this value/name.
```

Comment:
You are able to create snippets, that are using dynamic snippet names. This could be useful, if you only have a value/name stored in variable, but you need a translation of that value/name